content-src/reducers/SetRowsOrError.js

```diff
@@ -25,9 +25,15 @@ module.exports = function setRowsOrError(requestType, responseType) {
           state.init = true;
           state.rows = meta.append ? prevState.rows.concat(action.data) : action.data;
           state.error = false;
+          // If there is no data, we definitely can't load more.
           if (!action.data || !action.data.length) {
             state.canLoadMore = false;
           }
+          // If the results returned are less than the query size,
+          // we should be on our last page of results.
```

Comment:
is there going to be an edge case where we are on the last page of results but it is full?